### DISCUSSION TITLE
[V6] - active menu conflict in backpack-v6

### DISCUSSION THREAD

**Discussion by christmex** [Laravel-Backpack/community-forum#523] [General]
Hi guys, sorry for making so many questions 🙏 so I'm on a mission to test the backpack for laravel v6, and I found that the active menu have a conflict case when I have 2 or more menu that has the same prefix, let say I have, ``` employee ``` menu and ``` employee-position ``` menu when I visit the ``` employee-position ``` then the ``` employee ``` menu will set to active same as the current page that I visited which is ``` employee-position ``` menu, which they are not same.

<img width="469" alt="image" src="https://github.com/Laravel-Backpack/community-forum/assets/34405837/be3cbab9-c60c-4fb7-8109-73ef454282b4">

Here's my routes
<img width="594" alt="image" src="https://github.com/Laravel-Backpack/community-forum/assets/34405837/931a6ce4-15d0-4095-b67b-8f886bd4b15b">

Here's my ``` menu_items.blade.php ```
```php
<li class="nav-item dropdown">
    <a class="nav-link dropdown-toggle" href="#navbar-base" data-bs-toggle="dropdown" data-bs-auto-close="outside" role="button" aria-expanded="true">
    <span class="nav-link-icon d-md-none d-lg-inline-block"><!-- Download SVG icon from http://tabler-icons.io/i/package -->
        <svg xmlns="http://www.w3.org/2000/svg" class="icon" width="24" height="24" viewBox="0 0 24 24" stroke-width="2" stroke="currentColor" fill="none" stroke-linecap="round" stroke-linejoin="round" data-darkreader-inline-stroke="" style="--darkreader-inline-stroke: currentColor;"><path stroke="none" d="M0 0h24v24H0z" fill="none" data-darkreader-inline-stroke="" style="--darkreader-inline-stroke: none;"></path><path d="M12 3l8 4.5l0 9l-8 4.5l-8 -4.5l0 -9l8 -4.5"></path><path d="M12 12l8 -4.5"></path><path d="M12 12l0 9"></path><path d="M12 12l-8 -4.5"></path><path d="M16 5.25l-8 4.5"></path></svg>
    </span>
    <span class="nav-link-title">
        Employee
    </span>
    </a>
    <div class="dropdown-menu" data-bs-popper="static">
    <div class="dropdown-menu-columns">
        <div class="dropdown-menu-column">
            <a class="dropdown-item" href="{{ backpack_url('employee') }}"><i class="nav-icon la la-question"></i> Employees</a>
            <a class="dropdown-item" href="{{ backpack_url('employee-role') }}"><i class="nav-icon la la-question"></i> Employee roles</a>
            <a class="dropdown-item" href="{{ backpack_url('employee-position') }}"><i class="nav-icon la la-question"></i> Employee positions</a>
            <a class="dropdown-item" href="{{ backpack_url('employment-status') }}"><i class="nav-icon la la-question"></i> Employment statuses</a>
            <a class="dropdown-item" href="{{ backpack_url('employment-status-config') }}"><i class="nav-icon la la-question"></i> Employment status configs</a>
            <a class="dropdown-item" href="{{ backpack_url('employee-position-detail') }}"><i class="nav-icon la la-question"></i> Employee position details</a>
        </div>
    </div>
    </div>
</li>
```

so in my ``` menu_items.blade.php ``` I making changes in the class and HTML structure based on the tabler header structure, is that the mistake that I have made?

**Comment by munjaldevelopment** [comment]
@team, This issue happen on every version of Backpack (if we add same name slug start in menu). FYI

**Comment by fronbow** [comment]
This has been a problem for a while @christmex , from what I remember I think it's already on one of the brilliant team's TODO list!!

**Reply by christmex** [reply]
@fronbow right, thx for letting me know man.

Cheers

**Comment by karandatwani92** [comment]
Hey everyone,

Thanks for writing about it. I have created an issue on this and we will try to cover this soon. Updates can be followed here https://github.com/Laravel-Backpack/CRUD/issues/5114

Cheers

**Comment by pxpm** [comment]
This is fixed in Tabler 1.0.5 🙏

**Reply by christmex** [reply]
Great👏, thanks @pxpm and all of the team
